src/components/Utils/getCreateInventorySetting.js

```diff
@@ -0,0 +1,22 @@
+import { get } from 'lodash';
+
+const getCreateInventorySetting = (configs) => {
```

Comment:
sounds weird, and unclear what's going here without context

Reply:
it might be an option to add a comment describing what is `configs` and what function returns

Reply:
Added a comment